### PR TITLE
dolt-sql: add page

### PR DESCRIPTION
**Pull request by franzqat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by franzqat** [comment]
For Issue #5471

File: pages/common/dolt-sql.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# dolt sql
+
+> Run a SQL query. Multiple SQL statements must be separated by semicolons.
+> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-sql>.
+
+- Run a single query:
+
+`dolt sql -q "INSERT INTO t values (1, 3);"`
```
Comment (navarroaxel) [inline pages/common/dolt-sql.md:8]: We prefer long arguments if available
```suggestion
`dolt sql --query "{{INSERT INTO t values (1, 3);}}"`
```

File: pages/common/dolt-sql.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Run multiple SQL statements:
+
+`dolt sql <<SQL ALTER TABLE t ADD c INT; INSERT INTO t VALUES (1, 2, 1); SQL`
```
Comment (navarroaxel) [inline pages/common/dolt-sql.md:12]: This bash command isn't valid.

Comment (franzqat) [reply pages/common/dolt-sql.md:12]: I found the command in [this example](https://docs.dolthub.com/sql-reference/version-control/dolt-system-tables#dolt_merge_status)

Apologies if it doesn't work, I'll remove it and if someone has a better example is always welcome to add in the page.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**